apps/ecosystem/src/ecosystem.repository.ts

```diff
@@ -328,12 +329,157 @@ export class EcosystemRepository {
       where: {
         ...payload
       },
-      select:{
+      select: {
         ecosystemRole: true
       }
     });
 
   }
-    
-    
+
+  /**
+* Description: Get getAgentEndPoint by orgId
+* @param orgId 
+* @returns Get getAgentEndPoint details
+*/
+  // eslint-disable-next-line camelcase
+  async getAgentDetails(orgId: number): Promise<org_agents> {
+    try {
+      const agentDetails = await this.prisma.org_agents.findFirst({
+        where: {
+          orgId
+        }
+      });
+      return agentDetails;
+
+    } catch (error) {
+      this.logger.error(`Error in getting getAgentEndPoint for the ecosystem: ${error.message} `);
+      throw error;
+    }
+  }
+
+  /**
+     * Description: Get getAgentEndPoint by orgId
+     * @param orgId 
+     * @returns Get getAgentEndPoint details
+     */
+  // eslint-disable-next-line camelcase
+  async getEcosystemLeadDetails(): Promise<ecosystem_orgs> {
+    try {
+      const ecosystemRoleDetails = await this.prisma.ecosystem_roles.findFirst({
```

Comment:
There is function named `getEcosystemRole` in repository. we can reuse this

Reply:
ok